BowlingBall.Tests/GameFixture.cs

```diff
@@ -5,10 +5,109 @@ namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        RollingBall balls = new RollingBall();
+
+        [Fact]
+        
+        public void BasicTestForZeroScoring()
+        {
+            // Arrange
+            int actual;
+            int expected;
+
+            // Act
+            balls.RollMultipleTimes(20, 0);
+            actual = balls.GetScore();
+            expected = 0;
+
+            // Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+         public void TestAllOnes()
+        {
+            balls.RollMultipleTimes(1, 20);
+
+            var actual = balls.GetScore();
+            var expected = 20;
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void TestingOneSpare()
+        {
+            balls.RollingASpare(7);
+            var actual = balls.GetScore();
+            var expected = 10;
+            Assert.Equal(expected, actual);
+        }
+        [Fact]
+        public void TestingANumber()
+        {
+            balls.RollsNormal(4);
+            var actual = balls.GetScore();
+            var expected = 4;
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void testingAOneSpareOneStrike()
+        {
+            balls.RollingASpare(9);
+            balls.RollingAStrike();
+            var actual = balls.GetScore();
+            var expected = 30;
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void testingPerfectGame()

```

Comment:
use PascalCase for method name. for test name we generally follow convention like TestBowlingGameScore_AllStrikes_ShouldReturn300